packages/attestation-service/src/db.ts

```diff
@@ -20,3 +20,21 @@ export function initializeKit() {
     kit = newKit(fetchEnv('CELO_PROVIDER'))
   }
 }
+
+export async function existingAttestationRequest(
+  phoneNumber: string,
+  account: string,
+  issuer: string
+): Promise<AttestationStatic | null> {
+  const AttestationTable = await Attestation(sequelize!)
```

Comment:
Don't know anything about sequelize, just wondering if it'd be better to cache this AttestationTable object instead of recreating it? maybe a  non-issue

Reply:
I personally don't like global caching too much so eventually there should probably be a cache manager, but I figured for now it's no big deal

Reply:
If you but it above in the global scope, it's not actually global caching, it's module level caching, which is totally fine. If you don't export the cache var, it won't pollute your global scope. I think the bias against is mostly a holdover from browser days when the polluting the global scope was a big issue. 
But yeah really nbd, just a thought.

Reply:
I changed it to module level caching!